xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -968,16 +968,21 @@ bool CXBMCApp::GetExternalStorage(std::string &path, const std::string &type /*
 
 bool CXBMCApp::GetStorageUsage(const std::string &path, std::string &usage)
 {
-#define PATH_MAXLEN 50
+#define PATH_MAXLEN 38
 
   if (path.empty())
   {
     std::ostringstream fmt;
-    fmt.width(PATH_MAXLEN);  fmt << std::left  << "Filesystem";
-    fmt.width(12);  fmt << std::right << "Size";
-    fmt.width(12);  fmt << "Used";
-    fmt.width(12);  fmt << "Avail";
-    fmt.width(12);  fmt << "Use %";
+    fmt.width(PATH_MAXLEN);
+    fmt << std::left << "Filesystem";
+    fmt.width(12);
+    fmt << std::right << "Size";
+    fmt.width(12);
+    fmt << "Used";
+    fmt.width(12);
+    fmt << "Avail";
+    fmt.width(12);
+    fmt << "Use %";
```

Comment:
if you don't change any logic it is in general not necessary to change the formatting, as this usually just clutters the pr. still you can leave this change like it is now, but pretty pretty please add a few blank lines. it increases readability a lot:
```c
    fmt.width(PATH_MAXLEN);
    fmt << std::left << "Filesystem";

    fmt.width(12);
    fmt << std::right << "Size";

    fmt.width(12);
    fmt << "Used";

    fmt.width(12);
    fmt << "Avail";

    fmt.width(12);
    fmt << "Use %";
```

Reply:
I thought it was not necessary to add the blank lines, but I add them as you propose :)